package.json

```diff
@@ -2,24 +2,18 @@
   "name": "when-dom-ready",
   "version": "1.2.4",
   "description": "$(document).ready() for the 21st century",
-  "main": "dist/index.js",
+  "main": "dist/es5.umd/index.js",
+  "jsnext:main": "src/index.js",
```

Comment:
How interesting ! Thanks for the info. (We really profit when contributing to open source)